gapic/src/main/com/google/gapid/perfetto/views/TraceConfigDialog.java

```diff
@@ -512,7 +512,7 @@ public BasicInputArea(Composite parent, Settings settings, Theme theme,
 
         if (gpuCaps.getHasFrameLifecycle()) {
           gpuFrame = createCheckbox(
-              gpuGroup, "Frame Lifecycle", sGpu.getSurfaceFlinger(), e -> updateGpu());
+              gpuGroup, "Frame Lifecycle", false, e -> updateGpu());
```

Comment:
Default settings should be set in gapic/src/main/com/google/gapid/models/Settings.java.